05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+- Disadvantages: Compared to a simple random sample, for the same sample size, clustered samples generally have larger variance estimates.
+- Examples: 
+  - Example 1: Consider a study needing a sample of 6th grade students in the United States, no list likely exists of all these students. Obtaining a list of schools that have 6th graders is more likely to be possible so a study design could select a random sample of schools that have 6th graders. The selected schools can then provide a list of students to do a second stage of sampling where 6th grade students are randomly sampled within each of the sampled schools. This is a one-stage sample design and will be the type of design we will discuss in formulas below.
+  - Example 2: A more complicated but realistic example is necessary when sending out interviewers to households for a survey. Counties could be selected as the PSU, then Census block groups within counties selected as the secondary sampling unit (SSU), and then households within the block groups. This type of design reduces the travel necessary for interviewers.
+
+#### The math {-} 
+
+Consider a population where the are $N$ clusters and $n$ clusters are sampled via SRSWOR. Units within the sampled cluster are sampled via SRSWOR as well. Let $M_i$ be the number of units in cluster $i$ and $\bar{y}_i$ be the sample mean of cluster $i$. Then, a ratio estimator of the population mean is:
+
+$$\bar{y}=\frac{\sum_{i=1}^n M_i \bar{y}_i}{ \sum_{i=1}^n M_i}$$
+Note this is a consistent but biased estimator. Often the population size is not known so this is a method to estimate a mean without knowing the population size. The estimated standard error of the mean is:
+
+$$se(\bar{y})=\frac{1}{\hat{N}_{pop} } \sqrt{\frac{N^2 (1-\frac{n}{N})}{n}\frac{1}{n-1} \sum_{i=1}^n (M_i\bar{y}_i -\hat{t}/N)^2  + \frac{N}{n}  \sum_{i=1}^n \frac{M_i^2}{m_i}\left(1-\frac{m_i}{M_i}\right)s^2_i }$$
+where $\hat{N}_{pop}$ is the estimated population size, $\hat{t}$ is the estimated total, and $s_i^2$ is the sample variance of cluster $i$.
+
+For estimates of proportions, the estimated proportion is:
+
+$$\hat{p}=\frac{\sum_{i=1}^n M_i \hat{p}_i}{ \sum_{i=1}^n M_i}$$
+
+and the associated standard error estimate is:
+
+$$se(\hat{p})=\frac{1}{\hat{N}_{pop} } \sqrt{\frac{N^2 (1-\frac{n}{N})}{n}\frac{1}{n-1} \sum_{i=1}^n (M_i\hat{p}_i -\hat{t}/N)^2  + \frac{N}{n}  \sum_{i=1}^n \frac{M_i^2}{m_i}\left(1-\frac{m_i}{M_i}\right)s^2_i }$$
+
+where $s^2_i$ is defined as:
+
+$$s^2_i = \frac{m_hp_h(1-p_h)}{m_h-1}$$.
+
+#### The syntax {-} 
+
+To specify a two-stage clustered design without replacement,  use the following syntax:
+
+
+```{r, eval=FALSE}
```

Comment:
This code chunk has the old eval style.  Need to update to the quarto style like you have in the others.